.ci/validation.sh

```diff
@@ -201,7 +201,7 @@ no-error-xwiki)
   cd ..
   checkout_from https://github.com/xwiki/xwiki-platform.git
   cd .ci-temp/xwiki-platform
-  git checkout "1d243e9a31589c7a2b2b730bd5e""ad67cb989ca8b"
+#  git checkout "1d243e9a31589c7a2b2b730bd5e""ad67cb989ca8b"
```

Comment:
Do we still need to use this specific commit?

Reply:
If the external repo doesn't not enforce a stable build, we use specific commits that are stable. I thought xwiki was one of those.

Reply:
Updated.